.taskcluster.yml

```diff
@@ -0,0 +1,36 @@
+version: 0
+tasks:
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events:
+          - pull_request.closed
+          - pull_request.opened
+          - pull_request.reopened
+          - pull_request.synchronize
+          - pull_request.assigned
+          - pull_request.unassigned
+          - pull_request.labeled
+          - pull_request.unlabeled
+          - pull_request.edited
```

Comment:
I don't think we need all of these events -- probably just opened, reopened, and synchronize (which is when new changes get pushed)